schema/dataset-schema.md

```diff
@@ -0,0 +1,89 @@
+for **required** and `0,1` or `0+` for **optional** in the Cardinality column of the table below. 
+
+|Property|Class|Expected Type|Cardinality|Description|
+|---|---|---|---|---|
+|[`Schema:distribution`](https://schema.org/distribution)| Dataset| DataDownload | 1+ | A downloadable form of this dataset, at a specific location, in a specific format. This property can be repeated if different variations are available. There is no expectation that different downloadable distributions must contain exactly equivalent information (see also DCAT on this point). Different distributions might include or exclude different subsets of the entire dataset, for example.|
+|[`Schema:variableMeasured`](https://schema.org/variableMeasured)| Dataset | Text \| PropertyValue | 0+| The variableMeasured property can indicate (repeated as necessary) the variables that are measured in some dataset, either described as text or as pairs of identifier and description using PropertyValue. |
+|[`Schema:includedInDataCatalog`](https://schema.org/includedInDataCatalog)| Dataset | DataCatalog | 0,1 | A data catalog which contains this dataset. |
+
+The following examples demonstrate how each of these properties may
+be implemented in JSON+LD. 
+
+### Distribution
+
+The [`Schema:distribution`](https://schema.org/distribution) is a property of the [Schema:Dataset](https://schema.org/Dataset) class and must be expressed using the [`Schema:DataDownload`](https://schema.org/DataDownload) subtype. `Distribution` describes how the content related to the catalog record may be obtained. The `DataDownload` type contains a number of optional properties including `contentUrl`, `contentSize`, and `encodingFormat` that can be used to describe the downloadable content of the dataset.
+
+``` json
+{
+    "@context" : "https://schema.org/",
+    "@type" : "Dataset",
+    "name":"Fiber-optic distributed temperature sensing data collected in mine-impacted streams near Silverton, Colorado in September 2019 and 2021 (ver. 2.0, May 2022)",
+    "url":"https://www.sciencebase.gov/catalog/item/626b086bd34e76103cd183c5",
+    "identifier":"",
+    "description":"This dataset contains fiber-optic distributed temperature sensing (FO-DTS) data collected along the streambed interface of two streams named Cement Creek and California Gulch Creek, as well as the Animas River, located near Silverton Colorado. ",
+
+    ...
```

Comment:
Remove the ellipses  (...) from the document. These were used in the prior version but we decided to remove them in core.md and we should do the same here.

Reply:
Done!